src/gui/optionsdialog.ui

```diff
@@ -638,7 +638,7 @@
             <item>
              <widget class="QGroupBox" name="checkFileLog">
               <property name="title">
-               <string>&amp;Log file</string>
+               <string>&amp;Log File</string>
```

Comment:
Could you clarify why do you change some sentences to "Word1 word2 word3" while others to "Word1 Word2 Word3"? What rule do you follow?

Reply:
If the text is a header of some block/group then each word should be capitalized. Otherwise, lowercase (except proper nouns).
Here it is a header for a group.

Reply:
There's a size limit so technically it should be "Files".

Reply:
>There's a size limit so technically it should be "Files".

OK, done.